packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react'
+import { Router, Route } from 'react-router'
+
+import { LOCATION_CHANGE } from './reducer'
+
+class ConnectedRouter extends Component {
+  static propTypes = {
+    store: PropTypes.object,
+    history: PropTypes.object,
```

Comment:
My personal opinion as an end-user is to serve prefab exports, this avoids me on my end having to put a reference to the history module and init it and pass it in. It also means I dont need a dependency in my package to history (I know I could ignore it, but then some of my tools throw warnings) -- and personally you guys are better equipped to keep those dependencies updated along side react-router.

Just my 2¢